client/landing/stepper/declarative-flow/internals/steps-repository/import/helper.ts

```diff
@@ -45,6 +46,12 @@ export function getFinalImporterUrl(
 		} );
 	}
 
+	if ( migrateEntireSiteFlow ) {
+		importerUrl = addQueryArgs( importerUrl, {
```

Comment:
I thought weird to enhance this function to know about the "Migrate entire site", but I think it makes sensei since `client/landing/stepper/declarative-flow/site-setup-flow.ts` need to call this query string.

And after thinking more, I think migration is tied import somehow, so it could be used in other flows too.